main.go

```diff
@@ -17,10 +17,11 @@ import (
 func init() {
 	log.SetFormatter(&log.JSONFormatter{})
 }
-
+//TODO set Prefix by echo Router library?
 func main() {
 	app := cli.NewApp()
 	app.Author = "MEDIGO GmbH"
+	os.Setenv("PREFIX_ROUTE", "/laika") //Added a prefix as an environment variable
```

Comment:
Instead of reading the env var here directly, add it to the `app.Flags` configuration below. That way you can set also a default value, but anyway the default prefix should be no prefix, so it doesn't break our current deploy. Also we prefix env vars with `LAIKA_` so it doesn't clashes with other things running.